app/src/main/java/lt/vilnius/tvarkau/API/GetProblemsParams.java

```diff
@@ -0,0 +1,30 @@
+package lt.vilnius.tvarkau.API;
+
+import android.support.annotation.Nullable;
+
+public class GetProblemsParams {
+
+    private int start;
+    private int limit;
+    private String description_filter;
+    private String type_filter;
+    private String address_filter;
+    private String reporter_filter;
+    private String date_filter;
+    private String status_filter;
```

Comment:
Use `@SerializedName`. Also, there should be a naming policy for Gson to automatically map `type_filter <--> typeFilter`